.github/workflows/build.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       env:
         JAVA_TO_BUILD: ${{ matrix.version }}
         ARCHITECTURE: x64
+        CROSS_COMPILE: false
```

Comment:
Should that not be true?

Reply:
no this is for the normal (non cross compiled github actions builds